dali/operators/image/color/color_space_conversion.cc

```diff
@@ -22,42 +22,34 @@ DALI_SCHEMA(ColorSpaceConversion)
     .DocStr(R"code(Converts between various image color models.)code")
     .NumInput(1)
     .NumOutput(1)
-    .InputLayout("HWC")
-    .AddArg("image_type",
-        R"code(The color space of the input image.)code",
-        DALI_IMAGE_TYPE)
-    .AddArg("output_type",
-        R"code(The color space of the output image.)code",
-        DALI_IMAGE_TYPE);
+    .InputLayout({"FDHWC", "FHWC", "DHWC", "HWC"})
```

Comment:
I don't think we need them here if we allow more flexible layouts and check them programmatically in SetupImpl.

Reply:
As discussed offline, let's keep them as they provide documentation as well.